docs/operation/index.rst

```diff
@@ -0,0 +1,13 @@
+Operational Guide
+=================
+
+.. _MetalK8s: https://github.com/scality/metalk8s/
+
+This guide describes upgrade and downgrade guidelines, supported versions
+and best practices required for operating MetalK8s_. Refer to the
+:doc:`/installation-guide/index` if you do not have a working MetalK8s_ setup.
+
+.. toctree::
+
+   upgrade
+   downgrade
```

Comment:
You should remove downgrade section and trying to make the "upgrade.rst" more general IMO